src/workload_api/client.rs

```diff
@@ -319,16 +318,10 @@ impl WorkloadApiClient {
         &self,
         audience: T,
         jwt_token: &str,
-    ) -> Result<(SpiffeId, Option<Claims>), ClientError> {
```

Comment:
You might need to change the doc comment above to reflect the return type change